assets/css/src/newsletter_congratulate.styl

```diff
@@ -0,0 +1,19 @@
+.mailpoet_newsletter_loading
+  text-align: center;
+
+  .mailpoet_loading
+    margin: 100px auto 0 auto;
+
+  .mailpoet_newsletter_loading_header
+    margin: 30px;
+
+.mailpoet_congratulate_success
+  width: 100%;
+
+  h1
+    text-align center;
```

Comment:
<img width="1234" alt="screenshot 2018-10-31 16 46 38" src="https://user-images.githubusercontent.com/1082140/47801468-90a88f00-dd2e-11e8-9d6e-94aa97bd05cb.png">
The text is not centered properly. This happens because of the `Screen Options`  toggle is displayed on the right. It can be fixed by hiding the toggle or adding margin to `h1`

I would also suggest to add some bottom margin as well. The image seems to be to close to text - just my opinion.

I've noticed that also `loading dots` on previous screen were also affected same as the heading.